test-suite/rust-runtime-api/src/lib.rs

```diff
@@ -100,21 +99,34 @@ fn service_protos_with_service() {
 /// Rust-runtime API should return error in case of an incorrect artifact.
 #[test]
 fn service_protos_with_incorrect_service() {
+    use exonum_rust_runtime::_reexports::{ArtifactId, RuntimeIdentifier};
```

Comment:
Since `_reexports` module is private, I'd suggest to use exports from the `exonum` crate instead. I don't think adding `exonum` as a direct dependency is a big deal in this case.